### PR TITLE
cleanup repository

### PR DESCRIPTION
**Pull request by tfoote**
Remove wrapdocker, it's expected to come from outside. 
Switch to use librarian-puppet instead of manually installing the modules. 
Remove many unused files which are now in the buildfarm_deployment_config repo to avoid confusion. 

This needs to go with a matching buildfarm_deployment_config pull-request.

### CODE REVIEW

File: master/Puppetfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+forge "https://forgeapi.puppetlabs.com"
```
Comment (dirk-thomas) [inline master/Puppetfile]: Single quotes like the rest of the file?

File: repo/Puppetfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+forge "https://forgeapi.puppetlabs.com"
```
Comment (dirk-thomas) [inline repo/Puppetfile]: Single quotes like the rest of the file?

File: repo/Puppetfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+mod 'example42-iptables'
+mod 'garethr-docker'
+mod 'puppetlabs-ntp'
+mod 'rtyler/jenkins'
+mod 'tracywebtech-pip'
+
+
```
Comment (dirk-thomas) [inline repo/Puppetfile]: Remove second newline.

File: slave/Puppetfile [hunk 4]
```diff
@@ -0,0 +1,7 @@
+forge "https://forgeapi.puppetlabs.com"
```
Comment (dirk-thomas) [inline slave/Puppetfile]: Single quotes like the rest of the file?

File: master/Puppetfile (hunk 1 above)
Comment (tfoote) [inline master/Puppetfile]: I don't know the implications that's what the example looks like: https://github.com/rodjek/librarian-puppet

**Comment by dirk-thomas** [comment]
LGTM beside mini style comments.

**Pull request merged**